src/content/blog/starlight-035.mdx

```diff
@@ -0,0 +1,159 @@
+:::tip{icon="heart"}
+Show your love with a heart icon.
+:::
+```
+
+```mdx title="example.mdx"
+import { Aside } from '@astrojs/starlight/components';
+
+<Aside type="tip" icon="heart">
+	Show your love with a heart icon.
+</Aside>
+```
+
+```mdoc title="example.mdoc"
+{% aside type="tip" icon="heart" %}
+Show your love with a heart icon.
+{% /aside %}
+```
+
+Currently, asides can be styled using [Starlight’s built-in icons](https://starlight.astro.build/reference/icons/), but we’re planning to expand this to include [custom icon support](https://github.com/withastro/starlight/pull/3024) in a future release.
+
+Thank you to [Shubham Padia](https://github.com/shubham-padia) who contributed this feature!
+
+## Custom attributes for autogenerated sidebar links
+
+Starlight 0.35 adds support for applying custom attributes to all links in an autogenerated sidebar group.
+
+This makes it easier to add a custom style or behavior to all links in part of your sidebar.
+For example, you could decide to italicize all links in a specific autogenerated group:
+
+```js {9-10}
+starlight({
+	title: 'My docs',
+	sidebar: [
+		{
+			label: 'Constellations',
+			autogenerate: {
+				// Autogenerate a group of links for the 'constellations' directory.
+				directory: 'constellations',
+				// Italicize all link labels in this group.
+				attrs: { style: 'font-style: italic' },
+			},
+		},
+	],
+}),
+```
+
+## Custom page slug processing
+
+It’s now easier to customize how your docs file names are mapped to URLs.
+
+By default, Astro uses the popular [`github-slugger`](https://github.com/Flet/github-slugger) package to normalize content file names.
+For example, the content in `Example.File.md` is served at `/examplefile`.
+This ensures consistent output and URL compatibility with many common platforms.
+
+However, if you need to customize this normalization, you can now pass a [`generateId`](https://starlight.astro.build/reference/configuration/#generateid) option to Starlight’s `docsLoader()` in your content configuration.
+The following example avoids all normalization and just strips the file extensions from content files:
+
+```js {9-10}
+// src/content.config.ts
+import { defineCollection } from 'astro:content';
+import { docsLoader } from '@astrojs/starlight/loaders';
+import { docsSchema } from '@astrojs/starlight/schema';
+
+export const collections = {
+	docs: defineCollection({
+		loader: docsLoader({
+			// Remove the `.md` or `.mdx` extension, but otherwise don’t process filenames.
+			generateId: ({ entry }) => entry.split('.').slice(0, -1).join('.'),
+		}),
+		schema: docsSchema(),
+	}),
+};
+```
+
+## Bug fixes and more
+
+As always, we’ve also been working to fix issues and extend existing features. The [Starlight changelog](https://github.com/withastro/starlight/blob/main/packages/starlight/CHANGELOG.md) lists all the details including migration guidance for breaking changes and here are a few of the bigger changes in 0.35:
```

Comment:
```suggestion
As always, we’ve also been working to fix issues and extend existing features. The [Starlight changelog](https://github.com/withastro/starlight/blob/main/packages/starlight/CHANGELOG.md) lists all the details including migration guidance for breaking changes.

 Here are a few of the bigger changes in 0.35:
```
This was honestly the only place I stumbled a bit reading, so I think this might want to be two sentences? One option is to call a bit more attention to potentially breaking changes by putting this on a new line, but your call!

Reply:
Works for me! I semi wanted to avoid the `:` followed by headings, but the items also are kind of too large for a bullet list. I think you’re right here though.